### PR TITLE
Active directory

### PR DESCRIPTION
**Pull request by mjg59**

### CODE REVIEW

**Comment by mjg59** [comment]
https://github.com/coreos/bugs/issues/968

**Comment by mjg59** [comment]
Updated based on @crawford's review

File: net-fs/samba/samba-4.5.1.ebuild [hunk 1]
```diff
@@ -215,6 +208,11 @@ multilib_src_install() {
 		systemd_dounit "${FILESDIR}"/winbindd.service
```
Comment (crawford) [inline net-fs/samba/samba-4.5.1.ebuild]: I don't understand why I cannot leave a comment above this line...

We don't need the files installed on lines 201 and 202.

Comment (mjg59) [inline net-fs/samba/samba-4.5.1.ebuild]: Fixed.

File: net-fs/samba/samba-4.5.1.ebuild [hunk 2]
```diff
@@ -204,11 +204,16 @@ multilib_src_install() {
...
 		systemd_dounit "${FILESDIR}"/samba.service
 	fi
-	rm ${D}/usr/bin/ldb*
-	rm ${D}/usr/lib/debug/usr/bin/ldb*
-	rm ${D}/usr/lib64/samba/ldb/*
-	rm ${D}/usr/bin/tdb*
-	rm ${D}/usr/lib/debug/usr/bin/tdb*
+	rm ${D}/usr/lib*/samba/ldb/*
+	mv ${D}/usr/bin/net /tmp
```
Comment (marineam) [inline net-fs/samba/samba-4.5.1.ebuild]: Should this mv be inside `if use minimal`? Also, copping a simple name like "net" into and out of /tmp sounds dangerous. Use "${T}" instead which is local to the build and thus won't collide with anything.

File: profiles/coreos/base/package.use [hunk 3]
```diff
@@ -79,3 +79,5 @@ dev-cpp/glog gflags
 # enable rpc for rpc.rquotad
 sys-fs/quota rpc
 
+# don't ship the Samba daemons
+net-fs/samba minimal
```
Comment (marineam) [inline profiles/coreos/base/package.use]: Since this is a cusomized ebuild if you use `+minimal` in IUSE it'll turn on by default.

**Comment by mjg59** [comment]
@marineam Updated.

**Pull request merged**